README.md

```diff
@@ -185,6 +185,7 @@ This section is a great place to start if you want to get into improving OpenStr
 
 * [Back Of Your Hand](https://backofyourhand.com/) - A web map game that tests your knowledge by having you find a street in a given area. ([Source Code](https://github.com/adam-lynch/back-of-your-hand))
 * [PraxisMapper](https://github.com/drakewill-CRL/PraxisMapper) - GPS Game server, for making mobile games around exploring the real world. ([Source Code](https://github.com/drakewill-CRL/PraxisMapper))
+* [Arnis](https://github.com/louis-e/arnis) - Generate cities from real life in Minecraft using Python. ([Source Code](https://github.com/louis-e/arnis))
```

Comment:
For projects with only a GitHub link we omit the Source Code link, as duplicated links are bad practise.

```suggestion
* [Arnis](https://github.com/louis-e/arnis) - Generate cities from real life in Minecraft using Python.
```